site/content/en/docs/Concepts/services.md

```diff
@@ -27,7 +27,7 @@ When you're setting up a service, Copilot will ask you about what kind of servic
 
 If you want a service that can't be accessed externally, but only from other services within your application, you can create a __Backend Service__. Copilot will provision an ECS Service running on AWS Fargate, but won't set up any internet-facing endpoints.
 
-Currently there are a few service types supported:
+Currently these are the service types supported:
```

Comment:
This seems like a stylistic choice; it's a little more familiar, which I think is sort of the tone we're going for in our docs.